src/js/apps/patients/patient/sidebar/sidebar_app.js

```diff
@@ -8,7 +8,14 @@ import { SidebarView } from 'js/views/patients/patient/sidebar/sidebar_views';
 export default App.extend({
   onBeforeStart({ patient }) {
     const currentOrg = Radio.request('bootstrap', 'currentOrg');
-    const widgets = Radio.request('entities', 'widgets:collection', _.collectionOf(currentOrg.getSetting('widget_sidebar'), 'id'));
+    const widgetSettings = _.map(currentOrg.getSetting('widget_sidebar'), setting => {
```

Comment:
dan's suggestion is pretty interesting and it seems like it will work in this case:
If we add this to the widget collection in the entities.
```js
modelId(attr) {
    return _.uniqueId(`${ attr.id }-`);
},
```
This will allow us to add the same ids and have identical model copies..  it's the simplest solution for now.. if we ever have to `.get` or manipulate the sidebar live, we'll have to change it, but it seems like an improvement.. thoughts?

Reply:
Yeah just saw that suggestion. Agreed, it does sound much simpler.